dlang.org.ddoc

```diff
@@ -331,6 +331,7 @@ _=
 PC=$(TC p, $1, $+)
 _=
 
+$(COMMENT Should be used only when link text is not std.$1.$2 - use REF instead)
 PHOBOS=$(SPANC phobos, $(AHTTPS dlang.org/phobos/std_$1.html#$2, $(TAIL $+)))
```

Comment:
Let's simply get rid of this deprecated macro: https://github.com/dlang/dlang.org/pull/2195

Reply:
OK, reverted.